lib/shopify_cli/commands/app/create/rails.rb

```diff
@@ -5,7 +5,6 @@ class Create
         class Rails < ShopifyCLI::Command::AppSubCommand
           prerequisite_task :ensure_authenticated
 
-          recommend_default_node_range
```

Comment:
It seems that we do require Node to create Rails apps:
https://github.com/Shopify/shopify-cli/blob/35cb97a46e009c6f3fb5cc36321fc79cdb74c554/lib/shopify_cli/services/app/create/rails_service.rb#L116